integration_tests/integration_test.go

```diff
@@ -101,6 +101,47 @@ func TestKubernetesAndUpgradeOverlay(t *testing.T) {
 	RunCommandAndAssert(fmt.Sprintf("%s -n %s delete spinsvc %s", e.KubectlPrefix(), ns, SpinServiceName), t)
 }
 
+func TestUpdateSpinsvcStatus(t *testing.T) {
+	// setup
+	t.Parallel()
+	LogMainStep(t, `Test goals:
+- Install spinnaker
+- Check an OK status
+- Upgrade spinnaker with a bad configurations
+- Check a Failure status`)
+
+	spinOverlay := "testdata/spinnaker/base"
+	ns := RandomString("spinsvc-status")
+	e := InstallCrdsAndOperator(ns, true, defaults, t)
+	if t.Failed() {
+		return
+	}
+
+	// install
+	if !e.InstallSpinnaker(ns, spinOverlay, t) {
+		return
+	}
+
+	v := RunCommandAndAssert(fmt.Sprintf("%s -n %s get spinsvc %s -o=jsonpath='{.status.status}'", e.KubectlPrefix(), ns, SpinServiceName), t)
+	if t.Failed() || !assert.Equal(t, "OK", strings.TrimSpace(v)) {
```

Comment:
Out of curiosity, do we use an enum or similar type for SpinnakerService statuses in Operator?

Reply:
Oh, sorta, looks like it's defined in `spinnakerservice/status.go`, what if we used that here? I think the assertion should still hold

Reply:
sure thing, let me do that.